Source/Core/DolphinWX/MainMenuBar.cpp

```diff
@@ -463,11 +463,7 @@ wxMenu* MainMenuBar::CreateSymbolsMenu() const
   symbols_menu->Append(
       IDM_USE_SIGNATURE_FILE, _("Apply Signat&ure File..."),
       _("Must use Generate Symbols first! Recognise names of any standard library functions "
-        "used in multiple games, by loading them from a .dsy file."));
-  symbols_menu->Append(
-      IDM_USE_MEGA_SIGNATURE_FILE, _("Apply &MEGA Signature File..."),
-      _("Must use Generate Symbols first! Recognise names of any standard library functions "
-        "used in multiple games, by loading them from a .mega file."));
+        "used in multiple games, by loading them from a .dsy, .csv, .mega file."));
```

Comment:
an 'or' should likely be here before the last item in the list: `.dsy, .csv, or .mega file`